keps/sig-node/1977-container-notifier/kep.yaml

```
@@ -0,0 +1,48 @@
+title: Container notifier
+kep-number: 1977
+authors:
+  - "@xing-yang"
+  - "@yuxiangqian"
+owning-sig: sig-node
+participating-sigs:
+  - sig-storage
+status: implementable
```

To help this move forward, how about we try to merge it with
```suggestion
status: provisional
```

and then refine from there?

---

Sure.  We could try that.